README.adoc

```diff
@@ -65,7 +65,7 @@ Whether you use built-in repository interfaces or create custom repositories, Ja
 
 Jakarta Data supports two types of Pagination: offset and cursor-based Pagination.
 
-- **Offset Pagination**: Offset Pagination is a popular method for efficiently managing and retrieving large datasets. It is based on dividing the dataset into pages containing a specified number of elements. This method allows developers to retrieve a subset of the dataset by identifying the page number and the maximum number of elements per page.
+- **Offset Pagination**: Offset-based pagination relies on the LIMIT clause to tell the database to skip a certain number of rows before returning a specific number of rows.
```

Comment:
Users should not need prior knowledge of a LIMIT clause to understand our brief overview/introduction to pagination.  Here is a suggestion for some wording that will hopefully get the intended point across:

```suggestion
- **Offset Pagination**: Offset-based pagination computes pages relative to a positional offset from the beginning of the dataset given the requested page number and page size.
```